scripts/cluster-upgrade.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Variable sanity check
+if [[ -z ${PGVERSIONOLD} ]]; then
+  echo "Environment variable PGVERSIONOLD is empty."
+  echo "It must be set to postgresql version of the old cluster."
+fi
+if [[ -z ${PGVERSIONNEW} ]]; then
+  echo "Environment variable PGVERSIONNEW is empty."
+  echo "It must be set to postgresql version of the new cluster."
+fi
+if [[ -z ${PGDATAOLD} ]]; then
+  echo "Environment variable PGDATAOLD is empty."
+  echo "It must be set to the location of the old cluster."
+fi
+
+if [[ -z ${PGUNIXUSEROLD} ]]; then
+  PGUNIXUSEROLD=postgres
+fi
+
+if [[ -z ${PGDATANEW} ]]; then
+  PGDATANEW=/var/lib/postgresql/12/main
+fi
+
+# Inline replace
+# Change cluster data_directory
+sed -i 's|^data_directory|#data_directory|' /etc/postgresql/11/main/postgresql.conf
```

Comment:
Are you not supposed to use env variables to reference these so that this can be generic

Reply:
Ah, you mean the version. Yes, I forgot to replace it.